ingestion/src/metadata/ingestion/ometa/client_utils.py

```diff
@@ -28,7 +28,7 @@
 
 def create_ometa_client(
     metadata_config: OpenMetadataConnection,
-) -> OpenMetadata:
+) -> OpenMetadata[T, C]:  # pyright: ignore[reportInvalidTypeVarUse]
```

Comment:
Do you know if the using `# type: ignore` works as well?

Reply:
It is! https://docs.basedpyright.com/#/comments?id=line-level-diagnostic-suppression